Formula/openssl@1.1.rb

```diff
@@ -46,6 +47,13 @@ class OpensslAT11 < Formula
     end
   end
 
+  # Add darwin64-arm64-cc build target.
+  # See: https://github.com/openssl/openssl/pull/13476
+  patch do
+    url "https://github.com/felixbuenemann/openssl/commit/1b37aaa274811adde6052bfd9fc66dd370d2f4e5.patch?full_index=1"
```

Comment:
Please use https://github.com/openssl/openssl/pull/12369 instead, which is only awaiting policy clarification.

Reply:
No problem, will update ASAP.